sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Listeners/SharedBlobQueueListener.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.Azure.WebJobs.Extensions.Storage.Blobs.Listeners
 {
-    internal sealed class SharedBlobQueueListener : ISharedListener, IScaleMonitorProvider
+    internal sealed class SharedBlobQueueListener : ISharedListener, IScaleMonitorProvider, ITargetScalerProvider
```

Comment:
We need to validate manually if adding` ITargetScalerProvider` adds TBS for Blob trigger.

Reply:
Done, needed to add ITargetScalerProvider implementation in BlobListener